### ISSUE TITLE
H801 set color blinks once

### ISSUE THREAD

**Issue by mtmayr** [arendst/Sonoff-Tasmota#5211]
My H801 (20) works quite fine with firmware "Sonoff-Tasmota 6.4.1.8". Except for some reboots once a while. But when I upgraded to "6.4.1.15", setting a color results in showing the correct color, after 1s shortly showing another color (sometimes red, sometimes black, sometimes blue) in between for 2/10s, before showing the correct color again. It seems to be a part of the switching process. After that, the color is steady on an correct. No separate MQTT-message is sent. It seems to be dependant on the color, as some colors like #ff0000, #ffff00, #0000ff work fine, but e.g. #800050 does not. No complicated setup, just entering the color on console or via MQTT results in that behaviour!

Do you have any suggestions for me?

- [x] Searched the problem in issues and in the wiki : yes
- [x] Hardware used : H801 WiFi from AliExpress with MQTT
- [x] Development/Compiler/Upload tools used : Arduino 1.8.7 with Teensyduino 1.45
- [x] If a pre-compiled release or development binary was used, which one? : self compiled
- [x] You have tried latest release or development binaries? : src version: 6.4.1.15
- [x] Provide the output of command ``status 0`` :
```
12:03:29 MQT: stat/sonoff_33BC6868/STATUS = {"Status":{"Module":20,"FriendlyName":["Sonoff"],"Topic":"sonoff_33BC6868","ButtonTopic":"0","Power":1,"PowerOnState":3,"LedState":0,"SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0}}
12:03:29 MQT: stat/sonoff_33BC6868/STATUS1 = {"StatusPRM":{"Baudrate":115200,"GroupTopic":"sonoffs","OtaUrl":"http://192.168.1.21/firmware/sonoff.bin","RestartReason":"External System","Uptime":"0T00:00:22","StartupUTC":"2019-02-13T11:03:07","Sleep":50,"BootCount":2,"SaveCount":7,"SaveAddress":"F5000"}}
12:03:29 MQT: stat/sonoff_33BC6868/STATUS2 = {"StatusFWR":{"Version":"6.4.1.15(sensors)","BuildDateTime":"2019-02-13T12:00:37","Boot":31,"Core":"2_5_0_BETA2","SDK":"3.0.0-dev(c0f7b44)"}}
12:03:29 MQT: stat/sonoff_33BC6868/STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["xxx","xxx"],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","280500000100000000000000000000000000","00000000"]}}
12:03:29 MQT: stat/sonoff_33BC6868/STATUS4 = {"StatusMEM":{"ProgramSize":754,"Free":248,"Heap":16,"ProgramFlashSize":1024,"FlashSize":1024,"FlashChipId":"1440C8","FlashMode":3,"Features":["00000809","0FCEEB94","0007A3A5","B7FFBFCC","001ABBC0"]}}
12:03:29 MQT: stat/sonoff_33BC6868/STATUS5 = {"StatusNET":{"Hostname":"sonoff_33BC6868-7272","IPAddress":"192.168.1.78","Gateway":"192.168.1.1","Subnetmask":"255.255.255.0","DNSServer":"192.168.1.1","Mac":"80:7D:3A:33:BC:68","Webserver":2,"WifiConfig":5}}
12:03:29 MQT: stat/sonoff_33BC6868/STATUS6 = {"StatusMQT":{"MqttHost":"192.168.1.21","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_33BC6868","MqttUser":"socket","MqttType":2,"MAX_PACKET_SIZE":1000,"KEEPALIVE":15}}
12:03:29 MQT: stat/sonoff_33BC6868/STATUS7 = {"StatusTIM":{"UTC":"Wed Feb 13 11:03:29 2019","Local":"Wed Feb 13 12:03:29 2019","StartDST":"Sun Mar 31 02:00:00 2019","EndDST":"Sun Oct 27 03:00:00 2019","Timezone":"+01:00","Sunrise":"07:05","Sunset":"17:17"}}
12:03:29 MQT: stat/sonoff_33BC6868/STATUS10 = {"StatusSNS":{"Time":"2019-02-13T12:03:29"}}
12:03:29 MQT: stat/sonoff_33BC6868/STATUS11 = {"StatusSTS":{"Time":"2019-02-13T12:03:29","Uptime":"0T00:00:22","SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"POWER":"ON","Dimmer":50,"Color":"7F004F0000","HSBColor":"323,100,50","Channel":[49,0,30,0,0],"CT":499,"Scheme":0,"Fade":"OFF","Speed":1,"LedTable":"OFF","Wifi":{"AP":1,"SSId":"xxx","BSSId":"xx:xx:xx:xx:xx:xx","Channel":x,"RSSI":100}}}
```

**To Reproduce**
git clone
my_user_config.h: #define USE_CONFIG_OVERRIDE
my_user_config.h: #define FIRMWARE_CLASSIC
user_config_override.h: #define MODULE                 H801
user_config_override.h: increase CFG_HOLDER, 
flash H801 with soldered pins with Arduino IDE and jumper J3
disconnect+remove J3 and connect to external power
connect to web-interface/login
console command: "color #ff0000"
no blinking result, just red
console command: "color #800050"
fail result: displaying purple 1s, then black 2/10s, then stays purple

**Expected behavior**
just display #800050 without changing color
(it seems to work correctly with colors like #ff0000, #ffff00, #0000ff)

**Screenshots**
none, should I upload a video?

**Additional context**
Worked in 6.4.1.8, stopped working afterwards

**Comment by Jason2866** [comment]
Please erase flash via esptool.  Try latest precompiled binary from http://thehackbox.org/tasmota/
No OTA upload. Bad settings can be left in flash!
If error is still there provide `Status 0` output.

**Comment by mtmayr** [comment]
Thanks for reply Jason2866!
I tried to esptool erase_flash but that lead to an error: "ESP8266 ROM does not support function erase_flash."
When programming to thehackbox-version [1] you lead me, the color switching works fine!! ("esptool write_flash" erased the flash too I guess)
When programming again to my compiled version [2], it has the same unwanted behaviour again! Interesting!

[1] 6.4.1.16(b9f6bf4-sensors)-2_4_2
[2] 6.4.1.16(sensors)-2_5_0_BETA2
PS: I added the "status 0" above
